wazo_calld/plugins/switchboards/plugin.py

```diff
@@ -23,10 +27,17 @@ def load(self, dependencies):
         api = dependencies['api']
         ari = dependencies['ari']
         bus_publisher = dependencies['bus_publisher']
+        bus_consumer = dependencies['bus_consumer']
         config = dependencies['config']
         token_changed_subscribe = dependencies['token_changed_subscribe']
 
         confd_client = ConfdClient(**config['confd'])
+        switchboard_get_cache = ConfdClientGetUUIDCacheDecorator(confd_client.switchboards.get, 'switchboard')
+        confd_client.switchboards.get = switchboard_get_cache
+        line_get_cache = ConfdClientGetUUIDCacheDecorator(confd_client.lines.get, 'lines')
```

Comment:
Line only have ID, not UUID

Reply:
True, but the type of ID does not impact the `ConfdClientGetUUIDCacheDecorator`

Reply:
You're right, the cache invalidation won't work with the message body with key `id`.

Reply:
It's fixed.